fotoapparat/src/main/java/io/fotoapparat/hardware/v1/Camera1.java

```diff
@@ -433,4 +443,26 @@ private void recordMethod() {
         );
     }
 
+    @Override
+    public void setFocusArea(Rect cameraViewRect, float x, float y) {
+        Camera.CameraInfo info = getCameraInfo(cameraId);
+        SpaceConverter spaceConverter = new SpaceConverter(
+                displayOrientationDegrees,
+                info.facing == Camera.CameraInfo.CAMERA_FACING_FRONT
+        );
+
+        Rect previewSpaceRect = spaceConverter.previewToCamera(cameraViewRect, x, y);
+
+        List<Camera.Area> focusingAreas = new ArrayList<>();
+        focusingAreas.add(new Camera.Area(previewSpaceRect, FOCUS_METERING_AREA_WEIGHT_DEFAULT));
+
+        FocusUpdater focusUpdater = new FocusUpdater(
```

Comment:
Generally it is better to not create classes which do some logic inside methods. Instead, it should be passed as a constructor parameter. In this case I would say that this class should be a parameter of routine which uses it.